src/bio.c

```diff
@@ -2328,6 +2354,55 @@ int wolfSSL_BIO_flush(WOLFSSL_BIO* bio)
         return WOLFSSL_SUCCESS;
     }
 
+#ifdef OPENSSL_ALL
+    WOLFSSL_BIO* wolfSSL_BIO_new_ssl(WOLFSSL_CTX* ctx, int client)
```

Comment:
Does it make sense to have `wolfSSL_BIO_new_ssl_connect` call this function to generate the `SSL` `BIO`?

Reply:
Done